### ISSUE TITLE
[Feature Request] Filter boxes in References tab

### ISSUE THREAD

**Issue by FelesNoctis** [TES5Edit/TES5Edit#716]
#### _From @hazarduss in Discord_

Having filter options in the References tab similar to the ones in the View tab, in addition to the current sort function, would make searching large numbers of entries much simpler.
![filter box](https://i.imgur.com/9SForgY.png)

At the moment, the ability to navigate the references list is limited by only a single sort at a time, which can be a detriment in a large list. For instance: sorting by Name will alphabetize but mix the source plugins, and sorting by File will list the EDIDs by FormID, but you can't combine them.

Hazarduss's example talked about how mod authors will often try to organize their EDIDs, but what you're looking for may not necessarily be easier to find with sorting by Record as the keyword may be in the middle of the name. Selecting a certain record and then narrowing the [references list](https://cdn.discordapp.com/attachments/512837488922984458/597276366345469984/2019-07-06_205651.jpg) to, say, `temper` in relation to it would be much faster than needing to read through hundreds, or even thousands, of entries.

My example was to [look through this list](https://cdn.discordapp.com/attachments/512837488922984458/597300160778862604/0daePZg.png) and find `Seafoam`. You can, yes, but it would be **much** faster to be able to type in "Seafoam" and see a concise list.

As for specifc features, it could maybe include a dropdown box to be able to select what type of information you're filtering. Given the current category sort headers, being able to filter by Record name, Signature type, or plugin File would fit fairly well.

**Comment by CrazieIvan** [comment]
Hello,
The above activity on this request shows "ElminsterAU added this to the 4.1.x milestone on Oct 15, 2019".  Does that mean that I should be seeing this filter 4.1.3h build?  I'm not seeing it.  Do I need to enable it somehow?  

This is probably something very obvious and embarrassing :-)  Sorry if it's a dumb question.  
Thank you!

**Comment by VlitS** [comment]
Nope. All that means is that Elminster is planning on implementing the feature before the 4.2 release (which is likely still pretty far off). If it was implemented in a release, the issue would have a line saying "Elminster added a commit that referenced this issue", and possibly a comment about the specific version it will be in.

**Comment by CrazieIvan** [comment]
Aah OK, I appreciate the explanation!

**Comment by jbostrus** [comment]
I'm putting this in 4.1.5q 4921e339abc4d3891fa40697f5f11f955f760b5c
